src/switch_ivr_originate.c

```diff
@@ -1795,7 +1795,13 @@ SWITCH_DECLARE(switch_status_t) switch_ivr_enterprise_originate(switch_core_sess
 		if (cancel_cause && *cancel_cause > 0) {
 			handles[i].cancel_cause = *cancel_cause;
 		} else {
-			handles[i].cancel_cause = SWITCH_CAUSE_LOSE_RACE;
+            if ((hp!=NULL) && (hp->cause==SWITCH_CAUSE_SUCCESS)) { //Was this call taken by another destination?
```

Comment:
it's wrongly indented. Also the comments should be C-style: /* */

Reply:
let me know if its ok now

Reply:
Are you sure checking `hp`'s cause is right here?